src/ucp/rndv/rndv_put.c

```diff
@@ -605,6 +609,6 @@ ucp_proto_t ucp_rndv_put_mtype_proto = {
         [UCP_PROTO_RNDV_PUT_STAGE_ATP]        = ucp_proto_rndv_put_common_atp_progress,
         [UCP_PROTO_RNDV_PUT_STAGE_FENCED_ATP] = ucp_proto_rndv_put_common_fenced_atp_progress,
     },
-    .abort    = ucp_proto_abort_fatal_not_implemented,
+    .abort    = ucp_proto_rndv_ppln_abort,
```

Comment:
Not sure that current name of the function is a good one since `rndv/put/mtype` is not always a part of pipeline. Maybe rename to `ucp_proto_stub_abort`?

Reply:
ok, ucp_proto_rndv_stub_abort then